setup.py

```diff
@@ -11,6 +11,7 @@
     'scipy',
     'tabulate',
     'tensorboardX',
+    'pandas'
```

Comment:
Pandas is a fairly large dependency. Is it really necessary to add it?

Reply:
This is the idea I have got currently, I'm trying without it.